docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device.rst

```diff
@@ -92,21 +93,23 @@ CPU plugin supports the following floating-point data types as inference precisi
 
 - ``f32`` (Intel® x86-64, Arm®)
 - ``bf16`` (Intel® x86-64)
+- ``f16`` (Arm®)
 
-The default floating-point precision of a CPU primitive is ``f32``. To support the ``f16`` OpenVINO IR the plugin internally converts
+The default floating-point precision of a CPU primitive is ``f32``. On platforms that natively support half-precision calculations (``bfloat16`` or ``float16``),
+half-precision type (``bf16`` or ``f16``) is automatically used instead of ``f32`` to achieve better performance (see the `Execution Mode Hint <#execution-mode-hint>`__).
+Thus, no special steps are required to run a half-precision model.
+To support the ``f16`` OpenVINO IR on platforms that does not natively support ``float16`` the plugin internally converts
 all the ``f16`` values to ``f32`` and all the calculations are performed using the native precision of ``f32``.
-On platforms that natively support ``bfloat16`` calculations (have the ``AVX512_BF16`` or ``AMX`` extension), the ``bf16`` type is automatically used instead
-of ``f32`` to achieve better performance (see the `Execution Mode Hint <#execution-mode-hint>`__).
-Thus, no special steps are required to run a ``bf16`` model. For more details about the ``bfloat16`` format, see
-the `BFLOAT16 – Hardware Numerics Definition white paper <https://software.intel.com/content/dam/develop/external/us/en/documents/bf16-hardware-numerics-definition-white-paper.pdf>`__.
 
-Using the ``bf16`` precision provides the following performance benefits:
+Using the half-precision provides the following performance benefits:
 
 - ``bfloat16`` data type allows using Intel® Advanced Matrix Extension (AMX), which provides dramatically faster computations on corresponding hardware in comparison with AVX512 or AVX2 instructions in many DL operation implementations.
-- Reduced memory consumption since ``bfloat16`` data half the size of 32-bit float.
+- Reduced memory consumption since half-precision data half the size of 32-bit float.
```

Comment:
Reduced memory footprint since most of weight and activation tensors are stored in `half-precision`